### ISSUE TITLE
global image representations

### ISSUE THREAD

**Issue by TerryMelody** [ByChelsea/VAND-APRIL-GAN#31]
Hello dear authors！In the code "image_features, patch_tokens = model.encode_image(image, features_list)", is "image_features" the global image representations? Just like the source code in openclip: "image_features = model.encode_image(image)", as we always do. But you change the transformer to add additional outputs of assigned "features_list". I don't know whether my understanding is right or not.

**Comment by ByChelsea** [comment]
Yes, `image_features` is the global image representation.

By the way, you can use "copy permalink" to help others locate your issue more quickly. [Like this](https://github.com/ByChelsea/VAND-APRIL-GAN/blob/46fcbe567752dbcc84a26a05fb7b3794a72aa008/train.py#L112).
![image](https://github.com/ByChelsea/VAND-APRIL-GAN/assets/67221921/0dfaf267-8c61-4f5a-ac4d-a0bf9b7021f2)

**Comment by TerryMelody** [comment]
> Copy link
Thank you for your suggestions! Can I understand the “image features” in your code to be the same as those implemented in Openclip source codes. It seems you make a lot of modifications to openclip (https://github.com/mlfoundations/open_clip/tree/main/src/open_clip)?

**Comment by ByChelsea** [comment]
Yes, `image features` are the `class tokens` in the original implementation.

**Comment by TerryMelody** [comment]
> Yes, `image features` are the `class tokens` in the original implementation.

Thank you so much!
